### PR TITLE
Rename --no-loader-dupecheck option to --no-loader-dupe-check

### PR DESCRIPTION
**Pull request by magnumripper**
Also updates a couple of comments in john.conf to current option syntax

### CODE REVIEW

File: doc/OPTIONS [hunk 1]
```diff
@@ -50,14 +50,14 @@ this option would have the same result as having that word in the gecos
...
---single-retest-guess=BOOL
+--[no-]single-retest-guess
 
 Override config file setting for SingleRetestGuess.  Setting this to false
 stops Single mode from re-testing guessed plaintexts with all other salts.
 This is normally only needed for attacking massive, salted, input files
 eg. where you intend to use the --loopback option later instead.
 
---loader-dupecheck=BOOL
+--no-loader-dupecheck
```
Comment (solardiz) [inline doc/OPTIONS]: Forgot to insert the `-` here.

**Comment by solardiz** [comment]
Looks like this is based off `john.conf` prior to my addition of `PerRuleStats`, but GitHub seems happy to merge anyway.

**Review by solardiz — APPROVED** [review]

**Comment by magnumripper** [comment]
> Looks like this is based off `john.conf` prior to my addition of `PerRuleStats`, but GitHub seems happy to merge anyway.

That should be no problem unless diff contexts start clashing, but I rebased it now that I fixed the other thing.

**Comment by solardiz** [comment]
> unless diff contexts start clashing

I actually thought they could - it's nearby lines.

**Pull request merged**